tests/jax/integration/parameter_server_test.py

```diff
@@ -83,11 +83,14 @@ def test_parameter_server_single_process(test_system_sp: System) -> None:
             break
     assert at_least_one_changed
 
-    # run step function
+    #  Sleep until checkpoint_minute_interval elapses
+    time.sleep(parameter_server.store.checkpoint_minute_interval * 60 + 2)
```

Comment:
I think this would also neet to be fixed now due to the config update?

Reply:
This converts to seconds and then adds 2 seconds to make sure it goes into the if statement in the checkpointer

Reply:
I see. Thank you for calrifying :)